barretenberg/cpp/src/barretenberg/join_split_example/proofs/join_split/join_split.test.cpp

```diff
@@ -707,7 +707,7 @@ TEST_F(join_split_tests, test_0_input_notes_and_detect_circuit_change)
     // The below part detects any changes in the join-split circuit
     constexpr uint32_t CIRCUIT_GATE_COUNT = 49492;
     constexpr uint32_t GATES_NEXT_POWER_OF_TWO = 65535;
-    const uint256_t VK_HASH("cc3b14fad5465fe9b8c714e8a5d79012b86a70f6e37dfc23054e6def7eb1770f");
```

Comment:
changing a hardcoded hash should be fine here, but someone should double check me

Reply:
CI should automatically detect if you computed incorrectly